jira_scraper/config/settings.py

```diff
@@ -0,0 +1,15 @@
+"""Module for config settings."""
```

Comment:
This is pretty much just  file containing constants, we don't load any values from the outside.

Reply:
These are constants for configuration defaults atm. However, I'm sure we will evolve to load and parse data from different sources in the future by providing web UI. Then the config dataclass would be useful. Anyway, I can drop the class for the time being if that helps.